docs/auth-modes.md

```diff
@@ -0,0 +1,263 @@
+    </connection-resolver>
+    ...
+</tdr>
+```
+
+### Password only 
+
+User is prompted for password during initial connection creation and reconnecting to the data source.
+
+```xml
+<!-- Connection Dialog -->
+<connection-dialog class='sample'>
+    <connection-config>
+        <authentication-mode value='PasswordOnly' />
+        ...
+    </connection-config>
+</connection-dialog>
+```
+
+```xml
+<!-- Connection Resolver -->
+<tdr class='sample'>
+    <connection-resolver>
+        ...
+        <connection-normalizer>
+            <required-attributes>
+                <attribute-list>
+                    ...
+                    <attr>password</attr>
+                    ...
+                </attribute-list>
+            </required-attributes>
+        </connection-normalizer>
+    </connection-resolver>
+    ...
+</tdr>
+```
+
+### Username and Password
+
+User is prompted for username and password during initial connection creation, and password only when reconnecting to the data source.  If the username can be blank replace ```value='Basic'``` with ```value='BasicNoValidateFields'``` below.
+
+```xml
+<!-- Connection Dialog -->
+<connection-dialog class='sample'>
+    <connection-config>
+        <authentication-mode value='Basic' />
+        ...
+    </connection-config>
+</connection-dialog>
+```
+
+```xml
+<!-- Connection Resolver -->
+<tdr class='sample'>
+    <connection-resolver>
+        ...
+        <connection-normalizer>
+            <required-attributes>
+                <attribute-list>
+                    ...
+                    <attr>username</attr>
+                    <attr>password</attr>
+                    ...
+                </attribute-list>
+            </required-attributes>
+        </connection-normalizer>
+    </connection-resolver>
+    ...
+</tdr>
+```
+
+### Multiple Authentication Modes
+
+User is prompted for which authentication option to use, then a set of fields appear, conditional on that option.  Depending on the option selected the user may or may not be prompted for credentials when reconnecting to the data source.
+
+Support authention options are: 
+```
+None
+Username
+Password
+UsernameAndPassword
+LDAP
+```
+
+```xml
+<!-- Connection Dialog -->
+<connection-dialog class='sample'>
+    <connection-config>
+        <authentication-mode value='ComboBoxIntegrated' />
+        <authentication-options>
+            <option name="None" value="auth-none" />
+            <option name="Username" value="username" />
+            <option name="UsernameAndPassword" value="username-password" default="true" />
+        </authentication-options>
+        ...
+    </connection-config>
+</connection-dialog>
+```
+
+Note: the ```value``` attribute value for all options are customizable by connector author, except None, which is required to be ```auth-none```.  These option values are the persisted value of the authentication attribute in a Tableau Workbook (twb) or Tableau Data Source (tds) file. 
```

Comment:
are -> is  "...attribute value for all options is customizable by ..."  I'd also suggest that you lower-case "workbook" and "data source"